src/App/App.csproj

```diff
@@ -955,7 +960,10 @@
       <Version>1.3.7</Version>
     </PackageReference>
     <PackageReference Include="VpnSDK">
-      <Version>2.2.32</Version>
+      <Version>2.2.33</Version>
+    </PackageReference>
+    <PackageReference Include="VpnSDK.Private.OpenVpn">
+      <Version>1.0.37-develop00010</Version>
```

Comment:
Please change it to refer to 1.0.38 version instead

Reply:
Resolved